trafficFlow/carFollowingModel/lanes/baseLane.py

```diff
@@ -21,16 +21,23 @@ class BaseLane:
         place the vehicles in a default manner on the lane
     """
 
-    def __init__(self):
+    def __init__(self, full_length=1000.):
+        self.full_length = full_length
         self.number_of_vehicles = 0
```

Comment:
No need to explicitly store ``number_of_vehicles``, you can just use ``len(vehicles)``.